api/provider_api.md

```diff
@@ -24,22 +24,72 @@
 ## Test Drive
 
 Once the above are defined (and stuffed as environment variables),
-you are ready to user the [Provider API](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#/Providers/getProvidersByConsumerOrgId)
+you are ready to use API, confer its [`swagger`/`OpenApi` documentation](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html)
 
-To help you further, here is below a sample `curl` query that will `GET` the list of all the event providers you are entitled to use.
+To help you further, here are a few sample `curl` commands.
+ 
+The one below will `GET` the list of all the event providers you are entitled to use.
 
        curl -v --request GET \
-        --url ${api_url}/events/${consumerId}/providers \
+        --url https://api.adobe.io/events/${consumerId}/providers \
         --header "x-api-key: $api_key" \
         --header "Authorization: Bearer $jwt_token" \
         --header "Accept: application/hal+json"
         
-once you have the provider id of interest, you can list of the events (and event codes) 
-it supports by using another endpoint: [getProvidersById](https://www.adobe.io/apis/experienceplatform/events/ioeventsapi.html#/Providers/getProvidersById):  
-Here is the associated sample `curl` query 
+Now you have the provider ids, you can list their events metadata: 
 
       curl -v --request GET \
-        --url ${api_url}/events/providers/${providerId}?eventmetadata=true \
+        --url https://api.adobe.io/events/providers/${providerId}?eventmetadata=true \
         --header "x-api-key: $api_key" \
         --header "Authorization: Bearer $jwt_token" \
         --header "Accept: application/hal+json" 
+        
+To create your own [custom events provider](../using/custom_events.md) :
+
+    curl -v --request POST \
+      --url https://api.adobe.io/events/${consumerId}/${projectId}/${workspaceId}/providers \
+      --header "x-api-key: $api_key" \
+      --header "Authorization: Bearer $jwt_token" \
+      --header 'content-type: application/json' \
+      --header 'Accept: application/hal+json' \
+      --data '{
+          "label": "a label of your choice for you custom events provider",
```

Comment:
use consistent capitalization for `provider` (see next line). Suggest lower case

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/72bc6b24787508a87232097979d4137ae92cf07f